src/elements/logo/logo.md

```diff
@@ -0,0 +1,7 @@
+# CORE logo
+
+
+```jsx
+<Logo>Design</Logo>
```

Comment:
Please, add an example of usage with _Link_, it's more popular to have the logo in `<a>`